package/base-files/files/lib/functions/caldata.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
These are library files, which are non-executable, and thus should not contain a shebang.